lib/runner/request-helpers-presend.js

```diff
@@ -14,9 +14,10 @@ module.exports = [
         if (!context.item) { return done(new Error('Nothing to resolve variables for.')); }
 
         // @todo - resolve variables in a more graceful way
+
         // @todo - no need to sync variables when SDK starts supporting resolution from scope directly
-        context.item = new sdk.Item(context.item.toObjectResolved(null, [context.data,
-            context.environment.syncVariablesTo({}), context.collectionVariables.syncVariablesTo({}),
+        context.item = new sdk.Item(context.item.toObjectResolved(null, [context._variables.syncVariablesTo({}),
```

Comment:
@kamalaknn - wondering if we should make this resolution as part of utils? It is used in two primary places?

```
resolveProperty = function (prop, context) ...
```